easybuild/easyconfigs/l/leidenalg/leidenalg-0.8.2-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,46 @@
+easyblock = 'PythonBundle'
+
+name = 'leidenalg'
+version = '0.8.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/vtraag/leidenalg'
+description = """Implementation of the Leiden algorithm for various quality
+functions to be used with igraph in Python."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+builddependencies = [
+    ('pkg-config', '0.29.2'),
+    ('PyYAML', '5.3'),
+]
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('igraph', '0.8.2'),
+    ('python-igraph', '0.8.0'),
+]
+
+use_pip = False
+sanity_pip_check = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+exts_list = [
+    # Test-only dependency
+    ('ddt', '1.4.1', {
+        'checksums': ['0595e70d074e5777771a45709e99e9d215552fb1076443a25fad6b23d8bf38da'],
+    }),
+    (name, version, {
+        'buildopts': '--use-pkg-config',
```

Comment:
@zarybnicky the build step does not do anything when installing with pip (building occurs as part of the install step), so better to remove the buildopts here.

Reply:
@smoors `use_pip` is set to `False`, but `python setup.py build` is being run in the build step

Reply:
ah I missed that it was set to False. you can ignore my comment then.